CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+# How to contribute
+
+We're really glad you're reading this, because we need volunteer developers to help this project come to fruition.
+
+## Submitting changes
+
+Please send a [GitHub Pull Request to Trento](https://github.com/trento-project/web/pull/new/main) with a clear list of what you've done (read more about [pull requests](http://help.github.com/pull-requests/)). When you send a pull request, we will love you forever if you include updates and extensions to the Golang Tests. We can always use more test coverage. Please follow our coding conventions (below) and make sure all of your commits are atomic (one feature per commit).
```

Comment:
I think this reference to the Golang Tests are from the old frontend right?

Reply:
reference removed.